CONTRIBUTING.md

```diff
@@ -2,52 +2,75 @@
 
 ## Code of Conduct
 
-All interactions with this project follow our [Code of Conduct][code-of-conduct].
-By participating, you are expected to honor this code. Violators can be banned
-from further participation in this project, or potentially all Coinbase projects.
+All interactions with this project follow our [Code of Conduct][code-of-conduct]. By participating, you are expected to honor this code. Violators can be banned from further participation in this project, or potentially all Coinbase projects.
 
 [code-of-conduct]: https://github.com/coinbase/code-of-conduct
 
 ## Bug Reports
 
-* Ensure your issue [has not already been reported][1]. It may already be fixed!
-* Include the steps you carried out to produce the problem.
-* Include the behavior you observed along with the behavior you expected, and
-  why you expected it.
-* Include any relevant stack traces or debugging output.
+Before filing a bug report, ensure that your issue [has not already been reported][1]. It may already be fixed!
+
+If your bug hasn’t been fixed, follow these steps to file a bug report:
+
+1. If your issue is not reported or fixed, click the [*New*][5] button from the [*Issues*][1] page. A new interface will appear.
+2. Find _Bug report_ on the list, and click [_Get Started_][6]. A new interface will appear.
+3. Add a title for your bug report. It should briefly describe the problem.
+4. Follow the template that appears in the Write text box. This is the best way to describe the bug.
+* **Describe the bug** - A clear and concise description of what the bug is.
+* **To Reproduce** - Steps to reproduce the behavior.
+* **Expected behavior** - A clear and concise description of what you expected to happen.
+* **Additional context** - You can include any relevant stack traces or debugging output in this section.
```

Comment:
<img width="745" alt="Screen Shot 2021-12-06 at 11 17 02 AM" src="https://user-images.githubusercontent.com/642275/144908247-c12815e1-b6ea-4b42-a287-890e434f9a64.png">
Formatting seems a bit off in bulleted list under 4th point

Reply:
It should be indented properly - right now it's on the same indentation.

Reply:
Updated in newest commit.